.travis.yml

```diff
@@ -8,15 +8,15 @@ env:
 
 matrix:
   include:
-    # Most IDAs are Python 3.5 and Django 2.2. Test and run codecov on those.
-    - python: 3.5
+    # Most IDAs are Python 3.6 and Django 2.2. Test and run codecov on those.
```

Comment:
Are you sure about this change? I think most of them are still using python35 as we've changed most of the repos to use python35 & python38 and have dropped python36 unless necessary.

Reply:
I think you are right, but the zipp==3.1.0 requires python3.6, how should I proceed this then?

Reply:
constrain zipp to version <2.0.

Reply:
Thanks a lot @mzulqarnain1 , @UsamaSadiq ! I have reverted back to python 3.5 and put those constraints.

Reply:
@mzulqarnain1 and @UsamaSadiq , Can you please review this PR again?